src/plugins/index_pattern_management/public/plugin.ts

```diff
@@ -115,6 +122,73 @@ export class IndexPatternManagementPlugin
       },
     });
 
+    core.application.register({
+      id: IPM_APP_ID,
+      title: sectionsHeader,
+      status: core.chrome.navGroup.getNavGroupEnabled()
+        ? AppStatus.accessible
+        : AppStatus.inaccessible,
```

Comment:
what are the app statuses available? what does it mean to the UI?

Reply:
So it is an existing enum in core application service that:
1. AppStatus.accessible means the application will be visible in the left navigation.
2. AppStatus.inaccessbile means the application will not be visible in the left navigation and the application will show an error page when user manually type the url.